example/stack.yaml

```diff
@@ -0,0 +1,5 @@
+flags: {}
+packages:
+- '.'
+extra-deps: []
+resolver: nightly-2015-08-21
```

Comment:
> What about adding a stack version to this file?

@mbj I suppose the user could always specify this in their `stack.yaml`. I could provide it as an example and make sure it matches the stack version I am using. I probably won't pin it exactly, but rather I will have it match on the major and minor segments.